docs/source/api/rainmaker.rst

```diff
@@ -13,7 +13,6 @@ ESP RainMaker is an end-to-end solution offered by Espressif to enable remote co
 - RainMaker library (i.e. this library, to develop the firmware)
 - RainMaker Cloud (backend, offering remote connectivity)
 - RainMaker Phone App/CLI (Client utilities for remote access)
-
```

Comment:
Missing empty line is causing CI docs build to fail @VojtechBartoska. Please fix it @sanketwadekar

Reply:
Done